### ISSUE TITLE
Docker build fails

### ISSUE THREAD

**Issue by aniketmitra001** [heremaps/movetk#55]
The [workflow](https://github.com/heremaps/movetk/blob/master/.github/workflows/create-docker-image.yml) for creation of docker image fails with the following error

```Make  
/opt/conda/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: warning: libwebp.so.7, needed by /opt/conda/lib/libgdal.so, not found (try using -rpath or -rpath-link)```

**Comment by aniketmitra001** [comment]
```libwebp``` needs to be installed as a dependency from the conda-forge channel

**Comment by aniketmitra001** [comment]
Fixed with changes made to pull request #56(https://github.com/heremaps/movetk/pull/56)
